cdl_examples/tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803.cdl

```diff
@@ -0,0 +1,90 @@
+netcdf tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803 {
```

Comment:
This is just an example for a single month, but the real monthly files should be split into decades, starting with year 1, that is, this month would belong to a file named `..._198101-199012.nc`